rclrs/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rclrs"
-version = "0.2.0"
```

Comment:
Let's maybe do the version update in bulk for all packages.

Reply:
Pushed!

Reply:
@christopherjreid @nnmm This should be part of the release process in a separate PR, not part of an existing PR.